model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/AuthenticationSessionAdapter.java

```diff
@@ -45,19 +46,22 @@
 public class AuthenticationSessionAdapter implements AuthenticationSessionModel {
 
     private final KeycloakSession session;
-    private final RootAuthenticationSessionAdapter parent;
+    private final  SessionEntityUpdater<AuthenticationSessionEntity> updater;;
```

Comment:
nitpicking, to keep the diff easy to read, I would keep the `parent` field here and add the `updater` below.